gravitee-apim-e2e/jest.config.js

```diff
@@ -172,11 +172,11 @@ module.exports = {
 
   // A map from regular expressions to paths to transformers
   transform: {
-    '^.+\\.ts$': 'esbuild-jest',
+    '\\.(js|jsx|ts|tsx)$': '@sucrase/jest-plugin',
     '^.+\\.xml$': '<rootDir>/lib/jest-raw-loader.js',
   },
 
-  // testTimeout: 5000,
+  testTimeout: 30000,
```

Comment:
Is there a reason for this change ? 
If yes do we need to add it in the ci config too ?

Reply:
Yes, it's usefull on debug